R/fitting-and-postprocessing.R

```diff
@@ -64,7 +43,7 @@ extract_lognormal_draws <- function(data, id_vars, from_dt = FALSE) {
 draws_to_long <- function(draws) {
   long_draws <- data.table::melt(
     draws,
-    measure.vars = c("meanlog", "sdlog", "mean", "sd"),
+    measure.vars = c("mu", "sigma", "mean", "sd"),
```

Comment:
as you now have parameters called intercept etc does this not do weird things?

Reply:
(I don't think that I have parameters called intercept etc.)

Reply:
I can't see why you wouldn't given your update code.